### ISSUE TITLE
bottom bar fonts' size in webkit2 version

### ISSUE THREAD

**Issue by paoloap** [fanglingsu/vimb#332]
I know it's a minor issue, but I leave it here for when it will be in the development agenda... fonts into bottom bars (both status and input ones) are undersized. Here's my font section into config file:

```
set input-font-normal=Share Tech Mono 10
set input-font-error=Share Tech Mono 10
set status-font=Share Tech Mono 10
set status-ssl-font=Share Tech Mono 10
set status-sslinvalid-font=Share Tech Mono 10
```

But fonts seem to have '7' size. In fact, if I set them to '13' they come back to '10' size.

**Comment by fanglingsu** [comment]
Thank you for the report! You're right the font size is wrong now. The reason for this is, that the stylings are now applied via css to the gui elements and so the syntax has slighly be changed.  Vimb takes the font setting and prefixes it with 'font:' to generate the css rule for the gui element. What is allowd can be read [GTK font properties](https://developer.gnome.org/gtk3/stable/chap-css-properties.html#id-1.5.3.3.15). So `set input-font-normal=10pt "Share Tech Mono"` should work.
By the way, all these font settings are only for compatibility with the current vimb version and may be removed soon or later. If you want to be futore save, you should remove the `#define FEATURE_GUI_STYLE_VIMB2_COMPAT` from you src/config.h and set the whole css by yourself.
```
set input-css=background-color:#fff;color:#000;font:10pt monospace

set completion-css=...
set completion-hover-css=...
set completion-selected-css=...
set input-error-css=...
set status-css=...
set status-ssl-css=...
set status-sslinvalid-css=...
```

**Comment by rti** [comment]
Vimb is now using CSS to style the widgets. Putting a plain number as font size will be interpreted as `px`.

Try to append `pt` to the font size numbers. That should lead to the expected result. 

Also, for some reason I put my font size numbers in front. I think there was another font style string interpretation glitch when putting it at the end.

TL;DR

Try
```set input-font-normal=10pt Share Tech Mono
set input-font-error=10pt Share Tech Mono
set status-font=10pt Share Tech Mono
set status-ssl-font=10pt Share Tech Mono
set status-sslinvalid-font=10pt Share Tech Mono```
